game-core/src/main/java/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -815,6 +818,12 @@ void showMouseCursor() {
 
   private final class BackgroundDrawer implements Runnable {
     private volatile boolean running = true;
+    private final ExecutorService executor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors());
+
+    BackgroundDrawer(){
+      ((ThreadPoolExecutor)executor).setKeepAliveTime(2L, TimeUnit.SECONDS);
+      ((ThreadPoolExecutor)executor).allowCoreThreadTimeOut(true);
+    }
```

Comment:
I'm not 100% sure about the ExecutorService to use.
I decided to use a modified FixedSize Thread pool that shuts down its Threads when they are idling for more than 2 Seconds, because the Threads are really only used during initial loading and when changing the zoom setting